modules/client/src/main/scala/lucuma/itc/client/json/PosBigDecimalJson.scala

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.itc.client.json
+
+import eu.timepit.refined.types.numeric.PosBigDecimal
+import io.circe.*
+import io.circe.syntax.*
+
+implicit val EncoderPosBigDecimal: Encoder[PosBigDecimal] =
```

Comment:
We have some of these encoders defined in explore too
https://github.com/gemini-hlsw/explore/blob/master/common-graphql/src/main/scala/queries/schema/ITC.scala#L21

I wonder if we could make them common. Perhaps moving these to either `lucuma-schemas` or `core`?

I setup an itc lib but I'm not sure I want to add an extra slab to the dependency chain. but maybe it is worth it to share these encoders

Reply:
It does seem like for trivial ones like `PosBigDecimal` we could just add them to `core`?  For non-trivial ones I'm not sure.  Servers could expect different encodings in theory I suppose.  Should we require that wavelengths are always encoded in pm for example? 

At least for the ones that aren't directly tied to the ITC API, I'd be fine with moving them (all those in the `lucuma.itc.client.json` package).  For the API itself (e.g., `SpectroscopyModeInput`) it seems like maybe they should stay in the ITC client.  What do you think?